presto-parser/src/main/java/com/facebook/presto/sql/tree/CreateType.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.sql.tree;
+
+import org.apache.commons.lang3.tuple.Pair;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static com.google.common.base.Preconditions.checkArgument;
+import static java.lang.String.format;
+
+public class CreateType
+        extends Statement
+{
+    private final QualifiedName typeName;
+    private final Optional<String> distinctType;
+    private final List<Pair<String, String>> parameters;
```

Comment:
We try not to introduce dependencies in sql-parser because a lot of tools use our parser as library and introducing new dependencies might cause conflicts for dependencies. As in other places, we can just have separate lists of parameter names and parameter types instead to avoid introducing dependency on `Pair`.